velox/core/QueryConfig.cpp

```diff
@@ -43,7 +43,8 @@ CapacityUnit valueOfCapacityUnit(const std::string& unitStr) {
   if (unitStr == "B") {
     return CapacityUnit::BYTE;
   }
-  if (unitStr == "KB") {
+  // To be backward compatible this is lowercase.
+  if (unitStr == "kB") {
```

Comment:
Why don't we support all flavors of upper/lower cases?
Like tolower() it and then compare with lower case all time time?
@kewang1024 
And that's for all units.

Reply:
That's a good idea. Let me put that out in the followup PR? Looks like other folks are blocked by this issue for now.